requirements.txt

```diff
@@ -28,3 +28,7 @@
 
 # https://github.com/inveniosoftware/dictdiffer/pull/111
 -e git+https://github.com/inspirehep/dictdiffer.git#egg=dictdiffer
+
+# Latest Flower (0.9.2 is badly bugged).
+# See: https://its.cern.ch/jira/browse/INSPIR-1477
+-e git+https://github.com/mher/flower.git@9af065043fa200f41ca226795dc59ac7941ccc1a#egg=flower
```

Comment:
won't this install flower to all nodes (also web nodes), rather than only the worker nodes?